cmd/web-handlers.go

```diff
@@ -2199,6 +2199,12 @@ func toWebAPIError(ctx context.Context, err error) APIError {
 		return getAPIError(ErrBadDigest)
 	case IncompleteBody:
 		return getAPIError(ErrIncompleteBody)
+	case PrefixAccessDenied:
+		return APIError{
+			Code:           "AccessDenied",
+			HTTPStatusCode: http.StatusBadRequest,
+			Description:    err.Error(),
+		}
```

Comment:
```suggestion
		return getAPIError(ErrAccessDenied)
```

Reply:
ErrAccessDenied will remove the details that the user needs to know like 'Prefix Access is denied:bkt2/', which is given by err.Error().

Reply:
Thank you. Made this change.